docs/index.md

```diff
@@ -36,9 +36,11 @@ provider "timeplus" {
 
 ### Required
 
-- `api_key` (String, Sensitive) The API key to be used to call Timeplus API.
 - `workspace` (String) The ID of the workspace in which the provider manages resources.
 
 ### Optional
 
+- `api_key` (String, Sensitive) [Cloud] The API key to be used to call Timeplus Enterprise Cloud.
 - `endpoint` (String) The base URL endpoint for connecting to the Timeplus workspace. When it's not set, `https://us.timeplus.cloud` will be used.
```

Comment:
we should change default value to us-west-2

Reply:
updated